src/content/deployment/subctl/_index.en.md

```diff
@@ -95,7 +95,7 @@ deployment.
 <!-- markdownlint-disable line-length -->
 | Flag                               | Description
 |:-----------------------------------|:----------------------------------------------------------------------------|
-| `--cable-driver` `<string>`        | Cable driver implementation (defaults to strongSwan IPsec)
+| `--cable-driver` `<string>`        | Cable driver implementation; available options are `strongswan` (default), `libreswan` and `wireguard`
```

Comment:
```suggestion
| `--cable-driver` `<string>`        | Cable driver implementation. The available options are `strongswan` (default), `libreswan` and `wireguard`.
```

Reply:
> 

None of the other sentences in the table include a period at the end. This is true for all tables in that doc. We need to be consistent